com.unity.uiwidgets/Editor/UIWidgetsEditorPanel.cs

```diff
@@ -48,15 +50,44 @@ public class UIWidgetsEditorPanel : EditorWindow, IUIWidgetsWindow {
             D.assert(_wrapper != null);
             _wrapper?.Destroy();
             _wrapper = null;
-
+            _internalTextFonts.Clear();
             Input_OnDisable();
         }
+        
+        protected virtual void onFill() {}
+
+        List<UIWidgetsPanel.TextFont> _internalTextFonts = new List<UIWidgetsPanel.TextFont>();
+        protected void AddFont(string family, List<string> assets, List<int> weights) {
+            int repeat = 0;
+            for(int i = 0; i< _internalTextFonts.Count; i++ ) {
+                if (_internalTextFonts[i].family == family)
+                    repeat++;
```

Comment:
you can remove the variable repeat
the logic is the same if we change it to:
for (;;;) {
  if (...) { return; }

please refer to the conversation in wechat for a deeper analysis on this issue